### PR TITLE
Add allowsExternalPlayback Property for ios

### PR DESCRIPTION
**Pull request by genglei01**
Make the video control can disable allowsExternalPlayback for ios

### CODE REVIEW

File: ios/RCTVideo.m [hunk 1]
```diff
@@ -67,6 +68,7 @@ - (instancetype)initWithEventDispatcher:(RCTEventDispatcher *)eventDispatcher
     _controls = NO;
     _playerBufferEmpty = YES;
     _playInBackground = false;
+      _allowsExternalPlayback = YES;
```
Comment (cobarx) [inline ios/RCTVideo.m]: The indentation is off here.

**Review by cobarx — CHANGES_REQUESTED** [review]
Looks great. Can you add an entry in the README for this prop.

**Comment by genglei01** [comment]
I have updated the README.

**Review by cobarx — APPROVED** [review]

**Comment by cobarx** [comment]
Thanks for updating this!

**Pull request merged**